flytekit/models/security.py

```diff
@@ -0,0 +1,129 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import List, Optional
+
+from flyteidl.core import security_pb2 as _sec
+
+from flytekit.models import common as _common
+
+
+@dataclass
+class Secret(_common.FlyteIdlEntity):
+    class MountType(Enum):
+        ENV_VAR = _sec.Secret.MountType.ENV_VAR
+        """
+        Use this if the secret can be injected as an environment variable. Usually works for symmetric keys, passwords etc
```

Comment:
just a question: is this way of using multi-line strings as in-line comments a convention, or important for sphinx docs?

Reply:
Thats what I understand, that sphinx will convert this to a doc-string on the constant